lib/ssl/src/ssl_session.erl

```diff
@@ -207,3 +210,11 @@ record_cb(tls) ->
     tls_record;
 record_cb(dtls) ->
     dtls_record.
+
+legacy_session_id() ->
```

Comment:
Why is this function deleted from its previous place in the beginning of the file and then moved here?
I find it more natural that `legacy_session/0` and `legacy_session/1` are placed together.

Reply:
Because we keep exported function at the beginning of the file and local functions at the bottom.